src/msgflo.cpp

```diff
@@ -404,19 +408,19 @@ shared_ptr<Engine> createEngine(const EngineConfig config) {
             } else {
                 username = up;
             }
-            cout << "username: " << username << endl;
-            cout << "password: " << password << endl;
+            // cout << "username: " << username << endl;
```

Comment:
If it needs more testing, add some automated tests...
But yes, can log the flags also under the debug switch. Password should propably not be logged, too easy for it to be persisted somewhere it shouldn't.